Bottom: 95c89eddcb5f2a6753a5404db8da56b2f112602e
Top:    3686078f498a85257437c7271563f4069282f203
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-11 16:29:54 -0500

Move ansible role vars to role defaults

This allows other parts of the playbook i.e. host vars, command line the
ability to override these defaults when necessary.


---
diff --git a/roles/docker-openldap/defaults/main.yml b/roles/docker-openldap/defaults/main.yml
index 58247cf..473571b 100644
--- a/roles/docker-openldap/defaults/main.yml
+++ b/roles/docker-openldap/defaults/main.yml
@@ -28,3 +28,31 @@ openldap_backup_files:
 
 # Network names
 docker_network_frontend: 'frontend'
+
+
+# the name of the image being duilt and used for the container
+openldap_image_repo: https://github.com/novatechweb/docker-openldap.git
+openldap_image_name: '{{ docker_registry_username }}/ldap'
+
+# the name of the container being started
+openldap_container_name: '{{ docker_name_prefix }}ldap'
+
+# the name of the data-volume used by the container
+openldap_cv_name: '{{ docker_name_prefix }}ldap_CV'
+openldap_dv_name: '{{ docker_name_prefix }}ldap_DV'
+
+# restore directories to temporarly store data being restored into docker containers
+openldap_docker_backup_dir: '{{ docker_backup_dir }}/LDAP'
+host_openldap_docker_restore_dir: '{{ bacula_dest }}{{ openldap_docker_backup_dir }}'
+
+
+
+# *****************************************************************************
+# Utility container - phpldapadmin
+
+# the name of the image used for the container
+phpldapadmin_image_repo: https://github.com/AndrewCooper/docker-phpldapadmin.git
+phpldapadmin_image_name: '{{ docker_registry_username }}/phpldapadmin'
+
+# the name of the container being started
+phpldapadmin_container_name: '{{ docker_name_prefix }}phpldapadmin'
diff --git a/roles/docker-openldap/tasks/main.yml b/roles/docker-openldap/tasks/main.yml
index 125fc4f..55c3e11 100644
--- a/roles/docker-openldap/tasks/main.yml
+++ b/roles/docker-openldap/tasks/main.yml
@@ -66,12 +66,12 @@
     version: master
     dest: '{{ docker_projects_dir }}/docker-openldap'
 
-
 - name: build image
   docker_image:
     name: '{{ openldap_image_name }}'
     tag: '{{ docker_image_tag }}'
     path: '{{ docker_projects_dir }}/docker-openldap'
+    force: true
 
 # *****************************************************************************
 # Create the data volumes
diff --git a/roles/docker-openldap/vars/main.yml b/roles/docker-openldap/vars/main.yml
index f66b70d..ed97d53 100644
--- a/roles/docker-openldap/vars/main.yml
+++ b/roles/docker-openldap/vars/main.yml
@@ -1,28 +1 @@
 ---
-
-# the name of the image being duilt and used for the container
-openldap_image_repo: https://github.com/novatechweb/docker-openldap.git
-openldap_image_name: '{{ docker_registry_username }}/ldap'
-
-# the name of the container being started
-openldap_container_name: '{{ docker_name_prefix }}ldap'
-
-# the name of the data-volume used by the container
-openldap_cv_name: '{{ docker_name_prefix }}ldap_CV'
-openldap_dv_name: '{{ docker_name_prefix }}ldap_DV'
-
-# restore directories to temporarly store data being restored into docker containers
-openldap_docker_backup_dir: '{{ docker_backup_dir }}/LDAP'
-host_openldap_docker_restore_dir: '{{ bacula_dest }}{{ openldap_docker_backup_dir }}'
-
-
-
-# *****************************************************************************
-# Utility container - phpldapadmin
-
-# the name of the image used for the container
-phpldapadmin_image_repo: https://github.com/AndrewCooper/docker-phpldapadmin.git
-phpldapadmin_image_name: '{{ docker_registry_username }}/phpldapadmin'
-
-# the name of the container being started
-phpldapadmin_container_name: '{{ docker_name_prefix }}phpldapadmin'
diff --git a/roles/docker-openssl/vars/main.yml b/roles/docker-openssl/defaults/main.yml
similarity index 100%
rename from roles/docker-openssl/vars/main.yml
rename to roles/docker-openssl/defaults/main.yml
diff --git a/roles/docker-openssl/tasks/main.yml b/roles/docker-openssl/tasks/main.yml
index a3828f6..95601db 100644
--- a/roles/docker-openssl/tasks/main.yml
+++ b/roles/docker-openssl/tasks/main.yml
@@ -65,6 +65,7 @@
     name: '{{ openssl_image_name }}'
     tag: '{{ docker_image_tag }}'
     path: '{{ docker_projects_dir }}/docker-openssl'
+    force: True
 
 # *****************************************************************************
 # Create the data volumes
